### ISSUE TITLE
Making MyMeta Moar Like Designed

### ISSUE THREAD

**Issue by peth-yursick** [MetaFam/TheGame#175]
[MyMeta Design](https://www.figma.com/file/n36ggiRBQEGzRUbGxWfkMX/Meta-Game-(Copy)?node-id=58%3A2):
![image](https://user-images.githubusercontent.com/55857222/98441723-6e596100-2100-11eb-8b38-918bff827cce.png)

[MyMeta Currently](https://my.metagame.wtf/player/peth-yursick):
![image](https://user-images.githubusercontent.com/55857222/98441693-3f42ef80-2100-11eb-9ee5-b4bc02957ab4.png)

Let's disregard the custom backgrounds for now & focus on what's missing or misplaced:

Missing:
- Timezone.
- A container displaying NFTs.
- Allow people to "add new block".

Misplaced: 
- XP & Rank should be next to "MetaGame" in the membership section.
- The contact links (github & twitter) should be moved from below the name into its own container below the "about me" container.

**Comment by vanmoortel** [comment]
Hello 😄,
I just finished implementing the following things, I don't know where to do the pull request so I pushed on my git.
I didn't update the backend because I didn't have time to do it properly, so I put TODO where some things are missing.

- Add/Remove section
- Player Hero, rework design to match Designed
- Player Collab, fake data because I don't find the "role" and "timezone"(used "location" instead)
- Player Skills, rework design to match Designed
- Player Contact, rework design to match Designed
- Player Gallery, fake data
- Player Memberships, images was hardcoded (but should be store with ipfs?), I didn't find the information about XP for the other DAO
- Player Achievements, fake data

https://github.com/vanmoortel/TheGame/tree/feature/moar-like-designed

It was the first time I participated in such a horizontal organization so any criticism is more than welcome!
